js/Main.js

```diff
@@ -1,4 +1,24 @@
 let adventure_btn = document.getElementById("open_adventure_page")
+let HTML_account_name = document.getElementById("HTML_account_name")
+let HTML_account_money = document.getElementById("HTML_account_money")
+let HTML_account_gems = document.getElementById("HTML_account_gems")
+let HTML_account_level = document.getElementById("HTML_account_lvl")
+let HTML_level_progress_bar = document.getElementById("lvl_progress_bar")
+let HTML_account_exp_in_progress_bar = document.getElementById("HTML_account_exp")
+
+let retrievedObject = JSON.parse(localStorage.getItem("data"))
+let account_name_in_LS = retrievedObject.user_data.user_name
+let account_money_in_LS = retrievedObject.user_data.money
+let account_gems_in_LS = retrievedObject.user_data.gems
+let account_level_in_LS = retrievedObject.user_data.level
+let account_exp_in_LS = retrievedObject.user_data.experience
+
+HTML_account_name.outerText = account_name_in_LS
+HTML_account_money.outerText = account_money_in_LS
+HTML_account_gems.outerText = account_gems_in_LS
+HTML_account_level.outerText = `Level: ${account_level_in_LS}`
+HTML_account_exp_in_progress_bar.outerText = account_exp_in_LS
```

Comment:
There are quite a few announcements here. It doesn't look very pretty, but in principle it's a good approach. But I recommend doing the following
`document.getElementById("HTML_account_gems").outerText = retrievedObject.user_data.gems`
This must shorted your lines count